modules/geolocation/geolocation.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+
+class Geolocation:
+    """Locates the geographical position of a set of pixels"""
+
+    # TODO Class members
+    def __init__(self):
+
+        self.__cameraOrigin3o = np.array([0.0, 0.0, 2.0])
+        self.__cameraDirection3c = np.array([0.0, 0.0, -1.0])
+        self.__cameraOrientation3u = np.array([1.0, 0.0, 0.0])
+        self.__cameraOrientation3v = 1 * np.cross(self.__cameraDirection3c, self.__cameraOrientation3u)
+        self.__cameraResolution = np.array([1000, 1000])  # TODO Make global?
+        self.__referencePixels = np.array([[0, 0],
+                                           [0, 1000],
+                                           [1000, 0],
+                                           [1000, 1000]])
+
+        self.__pixelsToGeographicals = np.array([[[0, 0], [0, 0]],
```

Comment:
Are these the pixels we want to convert? If so, please give this a shorter name, I suggest pixelsToConvert

Reply:
Output of gather_point_pairs() (currently has placeholder values).